File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidsland.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

import java.util.List;

@JsonInclude(Include.NON_NULL)
@Data
public class Arbeidsland {

    private List<Arbeidssted> arbeidssted;

    private String harfastarbeidssted;

    private String land;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Arbeidsland.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Arbeidsland(
    var arbeidssted: List<Arbeidssted> = emptyList(),
    var bosted: ArbeidslandBosted? = null,
    var land: String? = null
)
